packages/framework/fluid-framework/api-report/fluid-framework.api.md

```diff
@@ -151,6 +157,12 @@ export type FlexList<Item = unknown> = readonly LazyItem<Item>[];
 // @public
 export type FlexListToUnion<TList extends FlexList> = ExtractItemType<ArrayToUnion<TList>>;
 
+// @public
+export enum FluidRuntimeMinVersion {
+    V1 = 0,
+    V2 = 1
+}
```

Comment:
Is there an advantage to using an enum instead of const object? We should try to avoid enums because they're a back-compat/type-compat problem. https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums